src/main/java/ama/post/PostController.java

```diff
@@ -0,0 +1,52 @@
+package ama.post;
+
+import ama.validation.Validator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+/**
+ * Created by Stephane on 2017-03-19.
+ */
+@Controller
+public class PostController {
+
+    @Autowired
+    private SubmissionPostRepository submissionPostRepository;
+
+    @Autowired
+    private Validator validator;
+
+    @GetMapping("/create_submission")
+    public String createAMASubmission(Model model) {
+        model.addAttribute("submissionPost",new SubmissionPost());
+        return "postsubmission";
```

Comment:
can we make 'postsubmission' and 'create_submission' the same string for consistency's sake?